modules/ocf/manifests/networking.pp

```diff
@@ -39,8 +39,12 @@
   } else {
     # Find the first network interface that starts with 'en', since those are
     # ethernet interfaces. (Won't work for the raspberry pi, since it uses wifi)
```

Comment:
Might want to update this comment

Reply:
I think it's still accurate, this won't work for the raspberry pi, since it uses wifi and this section is for ethernet, but it could be clarified.

Reply:
Or the comment can be moved down into some if block. And then we can add a similar one about the first network interface that starts with 'wl'